tests/attr/neuron/test_neuron_gradient_shap.py

```diff
@@ -53,9 +63,9 @@ def _assert_attributions(
             baselines = baselines(inputs)
 
         attrs_ig = []
-        for baseline in baselines:
+        for baseline in baselines.__iter__():
```

Comment:
is the `__iter__()` necessary here ?

Reply:
Whoops, good point, I was testing because this was causing type errors, but it was an issue with torch 1.3, will remove this, thanks!!